src/button/default.scss

```diff
@@ -2,38 +2,32 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 @import './button';
+@import './button-icon';
 @import '../notification-marker/index';
+@import './disabled';
 
 @mixin iui-button-default {
-  @include themed {
-    background-color: t(iui-color-background-1);
-    border-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-4));
-    color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
-  }
-
-  > .iui-icon {
-    @include themed {
-      fill: t(iui-icons-color-actionable);
-    }
-  }
+  @include iui-button-default-base;
 
-  &:enabled:hover,
-  &:enabled:active,
-  &:focus-visible {
+  &:hover,
+  &:active,
+  &:focus {
     @include themed {
       background-color: t(iui-color-background-1-overlay);
       border-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-1));
       color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-1));
     }
     @include iui-notification-marker-hover;
 
-    > .iui-icon {
-      @include themed {
-        fill: t(iui-icons-color-actionable-hover);
-      }
+    .iui-button-icon {
+      @include iui-button-icon(t(iui-icons-color-actionable-hover));
     }
   }
 
+  &:focus:not(:focus-visible) {
+    @include iui-button-default-base;
+  }
+
   &.iui-dropdown {
```

Comment:
Nit: this could be moved into a mixin similar to `iui-button-size` and included in `classes.scss` for consistency